src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -69,6 +84,7 @@ public static void setUpClass() throws SQLException, IOException, InvalidNamespa
         testDBName = TestUtils.generateNewDB();
         String dbConnectionUrl = String.format("jdbc:postgresql://localhost/%s", testDBName);
         testDataSource = TestUtils.createTestDataSource(dbConnectionUrl);
+        connection = testDataSource.getConnection();
```

Comment:
Does this need to be closed eventually?

Reply:
Added `connection.close()` to tear down.